src/main/java/io/swagger/codegen/languages/DefaultCodegenConfig.java

```diff
@@ -166,15 +166,24 @@ public void processOpts() {
 
         if (additionalProperties.containsKey(CodegenConstants.MODEL_PACKAGE)) {
             this.setModelPackage((String) additionalProperties.get(CodegenConstants.MODEL_PACKAGE));
+        } else {
+            // not set, use to be passed to template
+            additionalProperties.put(CodegenConstants.MODEL_PACKAGE, modelPackage);
```

Comment:
I don't think this could cause an error or something similar, but i'm wondering if you'd need to set modelPackage in additionalProperties if its value is null or empty.